### PR TITLE
re-import data at start of plotting

### PR DESCRIPTION
**Pull request by brownsarahm**
This makes the individual episodes a little more self-containted and makes it easier for instructors who may swap out between episdoes.  

It also will mean that we do not need to dis-allow this episode for teaching demos [see PR](https://github.com/carpentries/instructor-training/pull/1346/commits/5ecd1baf934afa0a386c74ee82e29027f7ed8d0c)

### CODE REVIEW

**Comment by ldko** [comment]
Note, this overlaps with #893 and #876. I think it would be good to get some sort of fix out for this as it clearly has come up for multiple people, but I am no longer a maintainer here. ping @maxim-belkin

**Comment by maxim-belkin** [comment]
Making episodes more self-contained is a good thing, so I'm +1 here. Following my comment from 2020 (https://github.com/swcarpentry/python-novice-inflammation/pull/876#issuecomment-706258785), I would suggest two things:
1. Put this in a prereq (or callout) box at the beginning of the episode.
2. Provide a brief explanation on when the proposed steps (importing numpy + loading data) are necessary.

**Comment by brownsarahm** [comment]
@maxim-belkin does that meet your request?

File: _episodes/03-matplotlib.md [hunk 1]
```diff
@@ -20,6 +20,20 @@ there is no official plotting library, `matplotlib` is the _de facto_ standard.
 import the `pyplot` module from `matplotlib` and use two of its functions to create and display a
 [heat map]({{ page.root }}/reference.html#heat-map) of our data:
 
+> ## Making sure we have the data
+>
+> If you are continuing inthe same notebook from the previous episode you already
+> have a `data` variable and have imported `numpy`.  If you are starting a new 
+> notebook at this point you need the following two lines.
```
Comment (maxim-belkin) [inline _episodes/03-matplotlib.md:27]:
```suggestion
> notebook at this point, you need the following two lines:
```

File: _episodes/03-matplotlib.md [hunk 2]
```diff
@@ -20,6 +20,20 @@ there is no official plotting library, `matplotlib` is the _de facto_ standard.
...
+>
+> If you are continuing inthe same notebook from the previous episode you already
+> have a `data` variable and have imported `numpy`.  If you are starting a new 
+> notebook at this point you need the following two lines. 
+>
+> ~~~
+> import numpy
+> data = numpy.loadtxt(fname='inflammation-01.csv', delimiter=',')
+> ~~~
+> {: .source}
```
Comment (maxim-belkin) [inline _episodes/03-matplotlib.md:33]:
```suggestion
> {: .language-python}
```

**Review by maxim-belkin — CHANGES_REQUESTED** [review]
Thanks, @brownsarahm. Looks good! A few minor fixes and it will be good to go.
I also wonder if this note should be a `.prereq` instead of a `.callout` -- in that case we could standardize the title to "Episode prerequisites" and later use this title in other episodes (if and when necessary).

**Comment by brownsarahm** [comment]
I do like episode prereqs. Espeically if we can promote that as a broader trend across lesson. Will change.

**Review by maxim-belkin — APPROVED** [review]
Thank you so much for this contribution, Sarah! Looks great!

**Pull request merged**